src/helpers/GoogleHelper.js

```diff
@@ -0,0 +1,14 @@
+import axios from 'axios';
+
+const responseGoogle = (response) => {
```

Comment:
The default export name is not the same as the name of the file

Reply:
@adesege worked on feedback